commit d4384cdf4729d520dbfbe75dcccbb0531d64203d
Author: Reto Gantenbein <reto.gantenbein@linuxmonk.ch>
Date:   Thu Feb 27 23:11:45 2020 +0100

    Revert "copy /etc/pki/ca-trust/extracted into chroot [GH#397]"
    
    This reverts commit 2be432603733ce19ee0467bb0534daefd5bf86ed.

diff --git a/mock/py/mockbuild/package_manager.py b/mock/py/mockbuild/package_manager.py
index 26ba65ae..db709b75 100644
--- a/mock/py/mockbuild/package_manager.py
+++ b/mock/py/mockbuild/package_manager.py
@@ -10,8 +10,6 @@ import time
 from textwrap import dedent
 from configparser import ConfigParser
 
-from distutils.dir_util import copy_tree
-
 from . import util
 from .exception import BuildError, Error, YumError
 from .trace_decorator import traceLog, getLog
@@ -291,15 +289,8 @@ Error:      Neither dnf-utils nor yum-utils are installed. Dnf-utils or yum-util
         for pki_file in glob.glob("/etc/pki/mock/RPM-GPG-KEY-*"):
             shutil.copy(pki_file, pki_dir)
 
-    @traceLog()
-    def copy_certs(self):
-        cert_path = "/etc/pki/ca-trust/extracted"
-        pki_dir = self.buildroot.make_chroot_path(cert_path)
-        copy_tree(cert_path, pki_dir)
-
     def initialize(self):
         self.copy_gpg_keys()
-        self.copy_certs()
         if self.buildroot.is_bootstrap:
             self.copy_distribution_gpg_keys()
         self.initialize_config()
